### ISSUE TITLE
Autonity contract tests in CI

### ISSUE THREAD

**Issue by b00ris** [clearmatics/autonity#190]
### Problem
We have Autonity contract(https://github.com/clearmatics/autonity/issues/166) tests implemented through truffle(https://github.com/clearmatics/autonity/pull/189). We need to run it in CI.

### Expected behaviour
Autonity contract tests are green in CI

**Comment by b00ris** [comment]
blocked by https://github.com/clearmatics/autonity/pull/185.
New CI is not merged into master.

**Comment by JekaMas** [comment]
Blocked by https://github.com/clearmatics/autonity/issues/191
